main.tex

```diff
@@ -1,846 +1,1374 @@
+
+A further known result~~\cite{kutner2004applied, rao1973linear,
+wakefield2013bayesian} gives an expression for
+\(\SSe\):
+
+\begin{align}\label{eqn:x_SSError_formula}
+    \SSe &= {\tilde{\textbf{p}}} ^ T \tilde{\textbf{p}}
+           \tilde{\textbf{p}} C \left(C ^ T C \right) ^ {-1} C ^ T
+           \tilde{\textbf{p}} \\
+    \SSe &={\tilde{\textbf{p}}} ^ T \tilde{\textbf{p}} - \tilde{\textbf{p}} C \textbf{x}^*
+\end{align}
+
+Using this approach, the memory-one representation \(\textbf{p}\in\mathbb{R}^4\) of any
+strategy against any other can can be measured and if
+(\ref{eqn:measured_condition_for_chi}) holds then (\ref{eqn:x_SSError_formula})
+can be used to identify if a strategy is acting extortionately. While the
+specific memory-one representation might not be one that acts extortionately or
+is even feasible (as noted in~\cite{Press2012}), a
+high \(\SSe\) does imply that a strategy is not extortionate. For a measured
+\(\textbf{p}\), \(\SSe\) corresponds to the best fitting \(\alpha, \beta\). Suspicion of
+extortion then corresponds to a threshold on \(\SSe\) and a comparison of the
+measured \(\chi=\frac{-\beta}{\alpha}\).
+
+\section*{Results}
+
+This section validates the approach of the previous section and
+present a number of numerical experiments to identify if strategies that perform
+strongly in evolutionary settings are close or not to the space of
+ZD strategies.
+
+\subsection*{Validation}
+
+To validate the method described, we use~\cite{Stewart2012} which
+presents results from a tournament with
+19 strategies
+with specific consideration given to ZD strategies. This
+tournament is reproduced here using the Axelrod-Python
+library~\cite{Knight2016}. To obtain a good measure of the corresponding
+transition rates for each strategy all matches have been run for
+2000 turns and every match has been
+repeated 60 times. All of this
+interaction data is available at~\cite{vincent_knight_2018_1297075}. Note that
+in the interest of open scientific practice,~\cite{vincent_knight_2018_1297075}
+also contains interaction data for noisy and probabilistic ending interactions
+which are not investigated here.
+
+Fig~\ref{fig:sserror_in_stewart_plotkin} shows the
+\(\SSe\) values for all the strategies in the tournament, as
+reported in~\cite{Stewart2012} the extortionate strategy Extort-2 gains a large number of
+wins. Notice that the mean \(\SSe\) for Extort-2 is approximately zero, while for
+the always cooperating strategy Cooperator the \(\SSe\) is far from zero. It is
+also clear that ZD-GTFT2 defined as a ZD strategy does not act
+extortionately. This is evident by the fact that it does not rank highly according
+to wins which is due to its value of \(\chi\) being less than 1.
+ZD-GTFT2 is referred to as a ``generous'' ZD strategy, other examples of this
+include ZDGen2 and ZDSet2 defined in~\cite{sep-prisoner-dilemma}. The
+general performance of these will be discussed in
+Section~\ref{sec:numerical-experiments}.

```

Comment:
This doesn't work any more since the sections are not numbered

Reply:
In Section Results?

Reply:
Yup, thought I'd caught all of these. Thanks!